examples/src/main/scala/org/apache/spark/examples/snappydata/WorkingWithJson.scala

```diff
@@ -87,7 +87,7 @@ object WorkingWithJson extends SnappySQLJob {
     morePeople.write.insertInto("people")
 
     //print schema of the table
-    println("Print Schema of the table\n################")
+    println("Print Schema of the table people\n################")
```

Comment:
Can be "Print schema of the People table"

Reply:
Noted.